setup/docker/compose.yml

```diff
@@ -0,0 +1,80 @@
+networks:
+  hostnet:
+    ipam:
+      driver: default
+      config:
+        - subnet: 10.5.5.0/24
+
+volumes:
+  pulsarconf:
+  pulsardata:
+  zkdata:
+  zklogs:
+  zkdatalog:
```

Comment:
anything expected here?

Reply:
for data persistence across compose up/down.